tasks.py

```diff
@@ -28,37 +31,40 @@
 
 
 @task
-def black(ctx, quiet=False):
+def black(ctx, target: t.Optional[str] = None, quiet=False):
```

Comment:
If adding types to these tasks, might as well add them to all the args and returns for consistency.

Reply:
Oh yeah, this looks bad, I didn't even realize I added types among non typed arguments :D

I rebased and updated the stub file to integrate the change in the other PR (lint task correctly told me it needed update, it is cool :))
And I typed all tasks fully.